cmd/admin-revoker/main.go

```diff
@@ -305,6 +564,79 @@ func main() {
 			fmt.Printf("%d: %s\n", k, revocation.ReasonToString[k])
 		}
 
+	case (command == "private-key-block" || command == "private-key-revoke") && len(args) == 2:
```

Comment:
`main` is already a hundred lines long, and this new case adds nearly a hundred more. It also adds pretty deep nesting: this case, then conditionals for dry-run, then conditionals for each command. Can some of this be broken out into additional helper functions which take `dryRun` as a parameter?

Reply:
Super valid. I definitely had a bit of tunnel vision here. I've gone ahead and broken these out into some helpers named after the sub-commands themselves. Main is now only about `26` lines longer.